recipes/bcm2835/build.sh

```diff
@@ -0,0 +1,12 @@
+set -ex
+
+export CPPFLAGS="-I$PREFIX/include $CPPFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+export CFLAGS="-g -O2 $CFLAGS"
```

Comment:
These flags should all be included in the default ones, except maybe `LD_LIBRARY_PATH` but that shouldn't be needed normally at all.

Reply:
I copied this from elsewhere of course but it's good to know that we do not have to set these explicitly :)